components/aws/sagemaker/tests/integration_tests/scripts/run_integration_tests

```diff
@@ -116,11 +116,16 @@ set -e
 function launch_eks() {
   echo "[Creating EKS] Launching EKS cluster $EKS_CLUSTER_NAME"
 
-  eksctl_args=( --managed --nodes "${EKS_NODE_COUNT}" --node-type=c5.xlarge --timeout=60m --region "${REGION}" --auto-kubeconfig --version "${EKS_CLUSTER_VERSION}" )
+  eksctl_args=( --managed --nodes "${EKS_NODE_COUNT}" --node-type=c5.xlarge --region "${REGION}" --version "${EKS_CLUSTER_VERSION}" )
   [ ! -z "${EKS_PUBLIC_SUBNETS}" ] && eksctl_args+=( --vpc-public-subnets="${EKS_PUBLIC_SUBNETS}" )
   [ ! -z "${EKS_PRIVATE_SUBNETS}" ] && eksctl_args+=( --vpc-private-subnets="${EKS_PRIVATE_SUBNETS}" )
 
-  eksctl create cluster "${EKS_CLUSTER_NAME}" "${eksctl_args[@]}" --enable-ssm
+  eksctl create cluster "${EKS_CLUSTER_NAME}" "${eksctl_args[@]}" --enable-ssm --dry-run | tail -n +2 > generated-cluster.yaml
```

Comment:
` eksctl create cluster "${EKS_CLUSTER_NAME}" "${eksctl_args[@]}" --enable-ssm --dry-run`
Prints a non yaml line, `tail -n +2` takes it out.